arduino/smartcar/smartcar.ino

```diff
@@ -1,9 +1,74 @@
 #include <Smartcar.h>
 
-void setup() {
-  // put your setup code here, to run once:
+const int fSpeed   = 50;  // 70% of the full speed forward
+const int bSpeed   = -50; // 70% of the full speed backward
+const int lDegrees = -90; // degrees to turn left
+const int rDegrees = 90;  // degrees to turn right
+const int goStraight = 0;
+
+
+ArduinoRuntime arduinoRuntime;
+BrushedMotor leftMotor(arduinoRuntime, smartcarlib::pins::v2::leftMotorPins);
+BrushedMotor rightMotor(arduinoRuntime, smartcarlib::pins::v2::rightMotorPins);
+DifferentialControl control(leftMotor, rightMotor);
+
+SimpleCar car(control);
+
+const int triggerPin           = 6; // D6
+const int echoPin              = 7; // D7
+const unsigned int maxDistance = 100;
+SR04 front{arduinoRuntime, triggerPin, echoPin, maxDistance};
+
+void setup()
+{
+    Serial.begin(9600);
+}
+
+void loop()
+{
+  const auto distance = front.getDistance();
+  // When distance is `0` it means there's no obstacle detected
+  if (distance > 0 && distance < 1000) {
```

Comment:
It's almost correct, but why `1000` as your max distance when on L20 you have set the `maxDistance` to be 100?
Can you use a more realistic number?

Reply:
It was a typo on my end, it is meant to be distance < 100.

Reply:
Waiting for your fixes so that this can be approved :+1: